tests/integration/net/simple/peer/with_create_test.go

```diff
@@ -314,7 +314,7 @@ func TestP2PCreate_WithP2PCollectionAndSubscription_ShouldSucceed(t *testing.T)
 				},
 			},
 			testUtils.CreateDoc{
-				NodeID: immutable.Some(0),
+				NodeID: immutable.Some(1),
```

Comment:
Question: what was the reason for this change?

Reply:
Debug artifact, reverted